src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/ActivatorUtilities.cs

```diff
@@ -15,8 +15,13 @@ namespace Microsoft.Extensions.DependencyInjection
     /// <summary>
     /// Helper code for the various activator services.
     /// </summary>
-    public static class ActivatorUtilities
+    public static partial class ActivatorUtilities
```

Comment:
Do we still need partial?

```suggestion
    public static class ActivatorUtilities
```